io.github.brainstormtrooper.facteur.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id" : "io.github.brainstormtrooper.facteur",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "44",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "io.github.brainstormtrooper.facteur",
+    "finish-args" : [
+        "--share=ipc",
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--socket=ssh-auth",
+        "--device=dri",
+        "--share=network",
+        "--filesystem=home:create",
+        "--filesystem=xdg-run/gvfsd",
+        "--filesystem=xdg-run/dconf",
+        "--talk-name=org.freedesktop.secrets",
+        "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
```

Comment:
still some dconf things left over. (lines 16 and 18)

Reply:
the app does use dconf to store settings, so don't I need access?

Reply:
use use dconf directly?

Reply:
removed